content/docs/learning-guide/tutorials/extend-the-editor/shape-example-cpp.md

```diff
@@ -468,9 +468,12 @@ In this example, the signal listener uses a lambda handler, so you can define th
 
 Define `CreateEntityWithShapeComponent(...)`, which communicates with O3DE EBuses to create a new entity with a component specified by the `typeId` parameter.
 
-1. Send a request to create a new entity by calling `EditorRequestBus::BroadcastResult(...)`. Dispatch the `EditorRequests::CreateNewEntity` event, which creates a new entity and returns its `AZ::EntityId`. The new entity's `AZ::EntityId` is stored in `newEntityId`.
+1. Send a request to create a new entity by calling `EditorRequestBus::BroadcastResult(...)`. Dispatch the `EditorRequests::IsLevelDocumentOpen` event, which will return true/false if a level has been loaded or not.
+   - Attempting to create an Entity without a level loaded will cause a crash, so if a level isn't loaded, we will print a warning message and then return.
```

Comment:
Should we fix the crash instead of documenting it? Trying to create an entity without a level loaded should just do nothing.

(Feel free to just open a backlog ticket)

Reply:
When this tutorial was originally created prefabs weren't on be default, but I just checked and it is indeed crashing with both prefabs enabled and disabled. I've created a backlog ticket here: https://github.com/o3de/o3de/issues/11832

Ideally yes the Editor shouldn't crash, but either way for the purposes of this sample tool it shouldn't try to create the new shape if a level isn't loaded since it won't be able to.

Reply:
Yeah I still wouldn't really recommend writing "the editor will crash if you do this" since it's a bad look, but you get my point :D